src/components/product/LoadingView.vue

```diff
@@ -32,13 +32,14 @@ export default {
   methods: {
     onClickRefreshButton() {
       const barcode = this.$route.params.code
-      FeedbackService.getFeedbackResult({ barcode: barcode })
-      .then(resp => (
+      FeedbackService.getFeedbackResult({
+        barcode: barcode,
+        user_id: typeof(this.$auth.user) !== "undefined" ? this.$auth.user.sub : ''
```

Comment:
Wouldn't it male sense to have this check if possible always in the same file? Like always in FeedbackService.js?

Reply:
I thought so too, but whatever I tried both checks have to happen. Not sure why but I had cases where either would fail at some point. But in general, I agree, to only do this in one spot

Reply:
I mean I guess I could try to find a fix for this, but this version was working reliable now